examples/03.Sensors/Piezo_Frequency/Piezo_Frequency.ino

```diff
@@ -26,24 +26,23 @@
    Tim Barrass 2013, CC by-nc-sa.
 */
 
+// increase the rate of updateControl from the default of 64, to catch the piezo's rapid transients
+#define MOZZI_CONTROL_RATE 150
```

Comment:
I had the idea that `CONTROL_RATE` had to be a power of two… I do not see at the top of my head why right now but we might change that to 128 or 256?

Reply:
I hesitated a second on that one, too. Strictly speaking, I guess the requirement to use only powers of two only really applies where the ratio of audio-/control-rate comes into play, somewhere. But yes, I guess you are right, and we simply should encourage something like this in the examples. Will adjust.